stripe/api_requestor.py

```diff
@@ -419,6 +456,9 @@ def interpret_streaming_response(self, stream, rcode, rheaders):
                     "can be consumed when streaming a response."
                 )
 
-            return self.interpret_response(json_content, rcode, rheaders)
+            self.interpret_response(json_content, rcode, rheaders)
+            raise RuntimeError(
```

Comment:
`-> Never` in interpret_response?

Reply:
`intepret_response` doesn't throw always, only if `self._should_handle_code_as_error(rcode)` which we have checked here.

Reply:
Added a comment in [Address code review](https://github.com/stripe/stripe-python/pull/1088/commits/067ab1bfd6a74acd7cb4bb8c4f7e878a2d0e4661)